lib/src/sdk.ts

```diff
@@ -450,3 +455,37 @@ function getSwapStep(error: Error): string {
 
   return "UNKNOWN_STEP";
 }
+
+async function decodePayloadAndPost(
+  binaryPayload: string,
+  beData: BEData,
+  providerId: string
+) {
+  const buyApiUrl =
+    "https://buy.api.aws.stg.ldg-tech.com/sell/v1/forgeTransaction/offRamp";
```

Comment:
This is stage environment, we need  to point prod. (Although it will be good find a good way to switch...)

Reply:
I added the prod url, after max confirmed that it works.

Reply:
You can check the code, there is a `customUrl` attribute already implement in SWAP for these cases